K8s-deployment/K8s-cluster/addons/mon-stack/install.sh

```diff
@@ -20,4 +21,5 @@ sleep 20
 helm install --version=6.16.10 -f grafana/grafana-values.yaml -f grafana/resource-values.yaml  grafana -n mon-stack   grafana/grafana
```

Comment:
Include correct updated helm verisons of charts for
- grafana
- prometheus
- loki
- promtail